### DISCUSSION TITLE
Request Integration to OKX wallet, support Darwinia/Crab chain by default

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#102] [Integration]
https://www.okx.com/web3

**Comment by nblogist** [comment]
Hey @hackfisher I looked around seems like we'll have to go the listing route, from a brief overview seems like the wallet supports the networks that are listed, do you want me to go the listing route?

**Comment by nblogist** [comment]
@1022ranji perhaps you can look into this, I'd probably need a Darwinia domain email to be able to apply for token listing: https://www.okx.com/token-listing-apply

**Reply by 1022ranji** [reply]
I'll request a Darwinia domain email like devrel@darwinia.network next Monday, and then you and @wuminzhe can move forward with this and anything else that requires an email submission. cc @nblogist

**Comment by nblogist** [comment]
![image](https://github.com/dcdao/discussion/assets/35498810/03b4e1f6-4253-4728-88b1-6598c79ec04f)

CC @1022ranji listing is mandatory!
